auth0/resource_auth0_organization.go

```diff
@@ -84,16 +82,16 @@ func newOrganization() *schema.Resource {
 }
 
 func createOrganization(d *schema.ResourceData, m interface{}) error {
-	o := expandOrganization(d)
```

Comment:
**question (non-blocking):** I'm noticing that many of the diffs in these PRs are replacements of single-character variables in favor of more descriptive name. Can this stylization be automated in some way? Perhaps in a lint or style check?

Reply:
Yep, as previously mentioned we'll focus on this after this epic:) Good point definitely 👍🏻